test/e2e_kubeadm/bootstrap_signer.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e_kubeadm
+
+import (
+	"k8s.io/kubernetes/test/e2e/framework"
+
+	. "github.com/onsi/ginkgo"
```

Comment:
we shouldn't do dot imports:
https://stackoverflow.com/questions/6478962/what-does-the-dot-or-period-in-a-go-import-statement-do#comment7619890_6478990

even if it's already part of the framework in mutliple locations.
the alternative is to call `ginkgo.MethodName`.
same for goomega

Reply:
That's currently the "standard" in e2e test (or at least the approach used in most of the places), and it seems also accepted by golang community specifically for testing https://github.com/golang/go/wiki/CodeReviewComments#import-dot

However, I'm fine also in changing this, but IMO switching to import without dot only in a few places might increase confusion. If this should be done, let's do this consistently across the E2E framework/in another PR

NB. If your goal is to make e2e test pass golint, please mind that there are also other non-conformance to be addressed (package naming, comments...)

Reply:
ok

Reply:
currently, test suites are not passing golint, and the same was for E2E kubeadm before this PR
IMO making this happen seems to be out of the scope of this PR...